src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+  const robotValues = Object.values(robot);
+  const robotKeys = Object.keys(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
```

Comment:
you can use `for in` loop for robot object
With `for in` Object.values and Object.keys unnecessary